### ISSUE TITLE
TypeScript 2.0: never type not detected by the Repl

### ISSUE THREAD

**Issue by vishallama** [HerringtonDarkholme/typescript-repl#15]
Repl can't seem to detect the `never` type (TypeScript 2.0).

For the following code snippet, obtained from [here](https://www.typescriptlang.org/docs/handbook/basic-types.html)

``` typescript
// Function returning never must have unreachable end point
function error(message: string): never {
    throw new Error(message);
}
```

The repl output is:
`Cannot find name 'never'`

**Comment by HerringtonDarkholme** [comment]
Yes, of course. Currently it only supports 1.8.

TS2.0 is officially released. I will add the support.

**Comment by vishallama** [comment]
Awesome! Thanks. Btw, I feel `tsun` is the best TypeScript Repl out there. Thank you, very much, for creating such a useful piece of software.

**Comment by HerringtonDarkholme** [comment]
I will close this issue when TS2.0 is supported :)

**Comment by HerringtonDarkholme** [comment]
I will close this issue when TS2.0 is supported :)

**Comment by HerringtonDarkholme** [comment]
Initial support for TS2.0 is pushed to master branch.
But there is still a lot more to do. However you can try it out.

**Comment by HerringtonDarkholme** [comment]
Version Bumped!

**Comment by vishallama** [comment]
The code snippet works! Thanks.
